src/main/java/mate/academy/Main.java

```diff
@@ -37,19 +57,39 @@ public static void main(String[] args) {
         MovieSession tomorrowMovieSession = new MovieSession();
         tomorrowMovieSession.setCinemaHall(firstCinemaHall);
         tomorrowMovieSession.setMovie(fastAndFurious);
-        tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
+        tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(ONE_DAY_OFFSET));
 
         MovieSession yesterdayMovieSession = new MovieSession();
         yesterdayMovieSession.setCinemaHall(firstCinemaHall);
         yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(ONE_DAY_OFFSET));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        User user = new User();
+        user.setPassword(USER_PASSWORD);
+        user.setEmail(USER_EMAIL);
+        userService.add(user);
+        System.out.println(user);
+
+        Optional<User> byEmail = userService.findByEmail(USER_EMAIL);
+        System.out.println(byEmail);
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+
+        User login = authenticationService.login(USER_EMAIL, USER_PASSWORD);
```

Comment:
The `login` method should authenticate an existing user by checking if the email exists and verifying the password. It should not create a new user if the email is not found. Please ensure the `login` method in `AuthenticationService` is implemented correctly according to the task requirements.